app/views/shared/nav/_nav.html.erb

```diff
@@ -0,0 +1,481 @@
+                  <%= render 'shared/nav/level_2_subcategory_link', text: 'What Works Fund', url: article_path('en', 'what-works-fund') %>
+                  <%= render 'shared/nav/level_2_subcategory_link', text: 'Funded Projects', url: article_path('en', 'what-works-grant-funded-projects') %>
+                  <%= render 'shared/nav/level_2_subcategory_link', text: 'FinCap Labs ', url: article_path('en', 'the-financial-capability-lab') %>
+                </ul>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </li>
+
+      <li class="nav__level-1__element" data-nav-level-1-item role="none">
+        <%= render 'shared/nav/level_1_heading', text: 'News &amp; Events' %>
+
+        <div class="nav__level-2" data-nav-level-2>
+          <div class="nav__level-2__wrap">
+            <p class="nav__level-2__description" role="category description">Get all the latest news, events and join in the conversation</p>
+
+            <ul class="nav__level-2__categories">
+              <li class="nav__level-2__heading">
+                <%= render 'shared/nav/level_2_heading', text: 'News &amp; Events' %>
+              </li>
+
+              <li class="nav__level-2__category">
+                <ul class="nav__level-2__subcategories" role="menu" aria-label="Events">
+                  <%= render 'shared/nav/level_2_subcategory_heading', text: 'Evaluate your programme', url: article_path('en', 'evaluate-your-programme') %>
+                  <%= render 'shared/nav/level_2_subcategory_link', text: 'Talk Money Week', url: '/en/talkmoneyweek' %>
+                </ul>
+              </li>
+
+              <li class="nav__level-2__category">
+                <ul class="nav__level-2__subcategories" role="menu" aria-label="Latest news">
+                  <%= render 'shared/nav/level_2_subcategory_heading', text: 'Latest News', url: 'http://fincap.org.uk/latest-news' %>
+                </ul>
+              </li>
+
+              <li class="nav__level-2__category">
+                <ul class="nav__level-2__subcategories" role="menu" aria-label="Blog">
+                  <%= render 'shared/nav/level_2_subcategory_heading', text: 'Blog', url: 'http://blog.fincap.org.uk/' %>
+                </ul>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </li>
+
+      <li class="nav__search-bar" data-nav-search-bar>
+        <%= render 'shared/search', location: 'navigation' %>
+      </li>
+
+      <li class="nav__close-button">
+        <a href="#" class="nav__close-button__text" data-mobile-nav-close>
+          Close
+          <svg xmlns="http://www.w3.org/2000/svg" class="nav__close-button__icon" focusable="false">
+            <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-icon--mobile-close-box"></use>
+          </svg>
+        </a>
+      </li>
+    </ul>
+  </div>
+</nav>
+
+<div class="mobile-nav-overlay" data-mobile-nav-overlay=""></div>
```

Comment:
Hi @davidtrussler I understand Venetia point of view. However I think we should make every one level your own partial.

## Suggestion 1

Transform each primary navigation into your own thing:

```
<%= render 'navigation/financial_capability' %>
<%= render 'navigation/about_strategy' %>
<%= render 'navigation/life_stages' %>
<%= render 'navigation/research_evaluation' %>
<%= render 'navigation/news' %>
```
This of course does not solve the duplication problem but it is easier to follow and understand what contain on each menu rather than have a big file. 

## Suggestion 2

Also I think you should move the render to methods on application helper like for example:

```ruby
module ApplicationHelper
  def render_level_2_heading(text:, url:)
    render 'navigation/level_2_subcategory_heading', text: text, url: url
  end

  def render_level_2_link(text:, url:)
    render 'navigation/level_2_subcategory_link', text: text, url: url
  end
end
```

The benefits will be seen by the next suggestion.

## Suggestion 3

Also for the subcategories level 2 and similars below:

```
<ul class="nav__level-2__subcategories" role="menu" aria-label="What Works Programme">
  <%= render 'shared/nav/level_2_subcategory_heading', text: 'What Works Programme', url: '/en/what-works/programme' %>
   <%= render 'shared/nav/level_2_subcategory_link', text: 'What Works Fund', url: article_path('en', 'what-works-fund') %>
   <%= render 'shared/nav/level_2_subcategory_link', text: 'Funded Projects', url: article_path('en', 'what-works-grant-funded-projects') %>
   <%= render 'shared/nav/level_2_subcategory_link', text: 'FinCap Labs ', url: article_path('en', 'the-financial-capability-lab') %>
</ul>
```
You can do something like:

```erb
   <% { 'What Works Programme' => '/en/what-works/programme',  
            'What Works Fund' => article_path('en', 'what-works-fund'), 
             'Funded Projects' => article_path('en', 'what-works-grant-funded-projects'),  
             'FinCap Labs ' =>  article_path('en', 'the-financial-capability-lab') 
           }.each do |text, link| %>
     <%= render_level_2_link(text: text, link: link) %>
   <% end %>
```

I am ok with above but if you can even make better by moving to a constant:

```erb
   <% SOME_CONSTANT.each do |text, link| %>
     <%= render_level_2_link(text: text, link: link) %>
   <% end %>
```

```ruby
module ApplicationHelper
   SOME_CONSTANT = { 
      'What Works Programme' => '/en/what-works/programme',  
       'What Works Fund' => article_path('en', 'what-works-fund'), 
      'Funded Projects' => article_path('en', 'what-works-grant-funded-projects'), 
       'FinCap Labs ' =>  article_path('en', 'the-financial-capability-lab') 
    }.freeze
    # ...
  end
```